### PR TITLE
0.9, docs.

### PR DESCRIPTION
**Pull request by atotic**
auth2 for calendar is buggy

### CODE REVIEW

File: google-calendar.html [hunk 1]
```diff
@@ -69,37 +64,37 @@
     is: 'google-calendar-list',
 
     properties: {
-      signedIn: {
+      _signedIn: {
```
Comment (ebidel) [inline google-calendar.html]: should this be private?

File: google-calendar.html [hunk 2]
```diff
@@ -69,37 +64,37 @@
...
        * A title to be displayed on top of the calendar list.
-       * @attribute title
-       * @type string
        */
       title: {
         type: String,
         value: 'My calendars'
       },
-      calendars: {
+      _calendars: {
```
Comment (ebidel) [inline google-calendar.html]: should this be private?

File: google-calendar.html [hunk 3]
```diff
@@ -107,14 +102,16 @@
      *
      * @method displayCalendars
      */
-    displayCalendars: function() {
-      if (this.signedIn && this.$.calendar.api) {
-        var request = this.$.calendar.api.calendarList.list({"key": ""});
+    _displayCalendars: function() {
```
Comment (ebidel) [inline google-calendar.html]: Not sure this should be  private

**Comment by ebidel** [comment]
Reviewed

File: google-calendar.html (hunk 2 above)
Comment (atotic) [inline google-calendar.html]: Who'd know? All this element does is display a list of calendars. It is more of a demo, less of a real calendar element. Real calendar would be displaying a schedule. Now that'd be awesome.

File: google-calendar.html (hunk 3 above)
Comment (atotic) [inline google-calendar.html]: I can't tell either. Just give the word on what the right thing to do is.

Comment (ebidel) [inline google-calendar.html]: I'd say anything that was public before should continue to be public. This method looks like that was the case.

File: google-calendar.html (hunk 2 above)
Comment (ebidel) [inline google-calendar.html]: The list of `.calendars` would be useful as a public property.

Comment (atotic) [inline google-calendar.html]: fixed.

File: google-calendar.html (hunk 3 above)
Comment (atotic) [inline google-calendar.html]: fixed.

**Comment by atotic** [comment]
fixes in.

**Comment by ebidel** [comment]
LGTM

**Pull request merged**